interpreter/go/go.go

```diff
@@ -64,6 +64,10 @@ func (g *goData) unref() {
 	}
 }
 
+func (g *goData) String() string {
```

Comment:
It feels like this additional function is not related to this PR.

apmint, golabels and ruby also do not have this function implemented.

Reply:
I can split this out and add this to all the missing places in separate PRs.